webapp-django/crashstats/topcrashers/tests/test_views.py

```diff
@@ -0,0 +1,223 @@
+import mock
+import pyquery
+from nose.tools import eq_, ok_
+
+from django.core.urlresolvers import reverse
+
+from waffle.models import Switch
+
+from crashstats.crashstats.tests.test_views import (
+    BaseTestViews, Response, mocked_post_123
+)
+from crashstats.supersearch.tests.common import (
+    SUPERSEARCH_FIELDS_MOCKED_RESULTS,
+)
+
+
+class TestViews(BaseTestViews):
+    base_url = reverse('topcrashers:topcrashers')
+
+    @staticmethod
+    def setUpClass():
+        TestViews.switch = Switch.objects.create(
+            name='topcrashers-real-time',
+            active=True,
+        )
+
+    @staticmethod
+    def tearDownClass():
+        TestViews.switch.delete()
```

Comment:
Don't these two need to call the parent setUpClass and tearDownClass ? 
Why are they static methods?!